README.md

```diff
@@ -35,13 +35,13 @@ Hence why nobody is allowed to push commits directly to the core. All changes ar
 
 #### Authenticity
 
-Fixing and implementing missing blizzlike content is one of our priorities, and we can boast to offer the most content-complete open source emulator.
+Fixing and implementing missing blizzlike content is one of our priorities, and we can boast to offer the most content-complete open-source emulator.
 <!-- Not sure for this below -->
-<!-- We were the first open source emulator to have almost every dungeon and raid working. -->
+<!-- We were the first open-source emulator to have almost every dungeon and raid working. -->
 
 #### Modules
 
-Modules are essential to AzerothCore's success. Modules allows users to plug them in and out easily, and do not require to modify the core files. It also means users can keep pulling the git changes from the main repository and only develop their own modules.
+Modules are essential to AzerothCore's success. Modules allow users to plug them in and out easily, and do not require to modify the core files. It also means users can keep pulling the git changes from the main repository and only develop their modules.
```

Comment:
"their own modules" is better to me (ofc it's me who wrote this lol)

Reply:
"Their own" is basically just repeating the same word two times since it means the same, it is a tautology